cypress/e2e/data-import/log-details/log-details-for-created-imported-order-records.cy.js

```diff
@@ -0,0 +1,148 @@
+    contributor: '100$a',
+    contributorType: 'Personal name',
+    productId: '020$a',
+    qualifier: '020$q',
+    productIDType: 'ISBN',
+    vendorReferenceNumber: '980$f',
+    vendorReferenceType: 'Vendor order reference number',
+    fundId: '981$b',
+    expenseClass: '981$c',
+    value: '100',
+    type: '%',
+    locationName: '049$a',
+    locationQuantityPhysical: '980$g',
+    volume: '993$a'
+  };
+  const actionProfile = {
+    typeValue: FOLIO_RECORD_TYPE.ORDER,
+    name: `C375178 Test Order ${getRandomPostfix()}`
+  };
+  const jobProfile = {
+    ...NewJobProfile.defaultJobProfile,
+    profileName: `C375178 Test Order ${getRandomPostfix()}`,
+  };
+
+  before('login', () => {
+    cy.loginAsAdmin();
+    // create mapping profile
+    cy.visit(SettingsMenu.mappingProfilePath);
+    FieldMappingProfiles.openNewMappingProfileForm();
+    NewFieldMappingProfile.fillPhysicalOrderMappingProfile(mappingProfile);
+    FieldMappingProfiles.saveProfile();
+    FieldMappingProfiles.closeViewModeForMappingProfile(mappingProfile.name);
+
+    // create action profile
+    cy.visit(SettingsMenu.actionProfilePath);
+    ActionProfiles.create(actionProfile, mappingProfile.name);
+    ActionProfiles.checkActionProfilePresented(actionProfile.name);
+
+    // create job profile
+    cy.visit(SettingsMenu.jobProfilePath);
+    JobProfiles.createJobProfile(jobProfile);
+    NewJobProfile.linkActionProfile(actionProfile);
+    NewJobProfile.saveAndClose();
+    JobProfiles.checkJobProfilePresented(jobProfile.profileName);
+
+    cy.createTempUser([
+      permissions.settingsDataImportEnabled.gui,
+      permissions.moduleDataImportEnabled.gui
+    ])
+      .then(userProperties => {
+        user = userProperties;
+
+        cy.login(userProperties.username, userProperties.password,
+          { path: TopMenu.dataImportPath, waiter: DataImport.waitLoading });
+      });
+  });
+
+  after('delete test data', () => {
+    Users.deleteViaApi(user.userId);
+    JobProfiles.deleteJobProfile(jobProfile.profileName);
+    ActionProfiles.deleteActionProfile(actionProfile.name);
+    FieldMappingProfiles.deleteFieldMappingProfile(mappingProfile.name);
+  });
+
+  it('C375178 Verify the importing of orders with pending status (folijet)',
+    { tags: [TestTypes.criticalPath, DevTeams.folijet] }, () => {
+      // TODO delete function after fix https://issues.folio.org/browse/MODDATAIMP-691
+      DataImport.verifyUploadState();
+      DataImport.uploadFile(filePathForCreateOrder, marcFileName);
+      JobProfiles.searchJobProfileForImport(jobProfile.profileName);
+      JobProfiles.runImportFile();
+      JobProfiles.waitFileIsImported(marcFileName);
+      Logs.checkStatusOfJobProfile(JOB_STATUS_NAMES.COMPLETED);
+      Logs.openFileDetails(marcFileName);
+      FileDetails.checkOrderQuantityInSummaryTable(quantityOfOrders);
+      FileDetails.checkSrsRecordQuantityInSummaryTable(quantityOfOrders);
+      ordersData.forEach(order => {
```

Comment:
please try to use 
cy.wrap(ordersData).each( order => {

Reply:
Done